cluster-autoscaler/cloudprovider/alicloud/alicloud_manager.go

```diff
@@ -107,7 +107,7 @@ func (m *AliCloudManager) GetAsgSize(asgConfig *Asg) (int64, error) {
 	if err != nil {
 		return -1, fmt.Errorf("failed to describe ASG %s,Because of %s", asgConfig.id, err.Error())
 	}
```

Comment:
```suggestion
	}
	if !sg.EnableDesiredCapacity {
	    klog.Warningf("The Expected Number of Instances of ASG %s is disabled", asgConfig.id)
	    return int64(sg.ActiveCapacity + sg.PendingCapacity), nil
	}
```

This is basically consistent with our code. 
But we have done more compatibility(*max*). If you don't want to, I think this is enough.

Reply:
Feel free to share your thoughts.